semantic_conventions/trace/database.yaml

```
@@ -205,10 +205,9 @@ groups:
         type: string
         requirement_level:
           conditionally_required: >
```

I think it the [Recommended](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-requirement-level.md#conditionally-required) attribute requirement level probably makes more sense now instead of [Conditionally Required](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-requirement-level.md#conditionally-required).

```suggestion
          recommended: >
```